src/routes/reference/jsx-attributes/use.mdx

```diff
@@ -34,3 +34,7 @@ declare module "solid-js" {
 	}
 }
 ```
+
+<Callout type="caution" title="Custom components">
+This directive does not currently work on custom components. [details here](https://github.com/solidjs/solid/discussions/722)
```

Comment:
> Directives only work with native HTML elements (HTML/SVG/MathML/Custom Elements). Directives are not forwarded and wont work in user defined components, such `<TextInput use:myinput={[..]}/>` .  
> [See also](https://github.com/solidjs/solid/discussions/722).

I reversed the explanation, because its more clear that way. At first, I thought it was saying that doesn't work with custom elements. I'd also change the title of the callout to "Limitations"

Thanks!

Reply:
Thanks!